docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -33,58 +33,77 @@
+on the labeled dataset, but keeping the backbone frozen.
+As the embeddings as input to the classification head are already computed,
+using them directly saves computation time.
 
 What you will learn
 -------------------
 * You learn how an active learning loop is set up and which components are needed for it.
 * You learn how to perform active learning with Lightly.
 
-Requirements
+Define your dataset
 ------------
-- Make sure you are familiar with the command line tools described at :ref:`lightly-command-line-tool`.
 - To make the definition of the dataset for training the classifier easy,
   we recommend using a dataset where the samples are grouped into folder according to their class.
   We use the clothing-dataset-small. You can download it using
 
 .. code::
 
-    git clone https://github.com/alexeygrigorev/clothing-dataset-small.git
+    git clone https://github.com/alexeygrigorev/clothing-dataset-small.git ./clothing-dataset-small
+
+The dataset's images are RGB images with a few hundred pixels in width and height. They show clothes
+from 10 different classes, like dresses, hats or t-shirts. The dataset is already split into a train,
+test and validation set and all images for one class are put into one folder.
+
+.. image:: ./images/clothing-dataset-small-structure.png
+  :width: 400
+  :alt: The directory and file structure of the clothing dataset small
 
 
 Creation of the dataset on the Lightly Platform with embeddings
 ---------------------------------------------------------------
 
-To perform samplings, we need to perform several steps, ideally with the CLI.
-More documentation on each step can be found here: :ref:`lightly-command-line-tool`.
-
-A. Train a model on the dataset, e.g. with
+To perform samplings, we need a dataset with embeddings on the Lightly platform.
+The first step for it is training an embedding model. Next, your dataset is embedded.
+Last, the dataset and the embeddings are uploaded to the Lightly platform.
+These three steps can all be done using our lightly pip package with the single lightly-magic command:
 
 .. code::
+    #### Commands in terminal / command line
 
-    lightly-train input_dir="/datasets/clothing-dataset-small/train trainer.max_epochs=5
+    # Lightly is installed as a pip package
+    pip install lightly
 
-B. Create embeddings for the dataset, e.g. with
+    # Your personal token for accessing the Lightly Platform is defined. You can get it from
+    # the Lightly platform at https://app.lightly.ai under your username and then Preferences.
+    export LIGHTLY_TOKEN="YOUR_TOKEN"
 
-.. code::
-    
-    lightly-embed input_dir="/datasets/clothing-dataset-small/train" checkpoint=mycheckpoint.ckpt
+    # You set the name your dataset should have on the lightly platform.
+    # You can give it any name you want and change the default name.
+    export LIGHTLY_DATASET_NAME="clothing_dataset_small"
```

Comment:
Would drop this part and just hardcode the name of the dataset to 'active learning clothing data' or something like this. I would try to minimize the use/ set of environment variables and reduce the complexity for the user as much as possible.

Reply:
I agree with @IgorSusmelj. We should find the simplest solution possible for the user...